list/schema/attribute.go

```diff
@@ -9,6 +9,13 @@ import (
 
 // Attribute define a value field inside the Schema. Implementations in this
 // package include:
+//   - BoolAttribute
+//   - Float32Attribute
+//   - Float64Attribute
+//   - Int32Attribute
+//   - Int64Attribute
```

Comment:
One extra attribute we'd want to support in the context of Terraform "numbers" is a `NumberAttribute`, which operates on a `*big.Float`